src/test/scala/org/apache/mesos/chronos/scheduler/jobs/ScheduleStreamSpec.scala

```diff
@@ -12,17 +12,17 @@ class ScheduleStreamSpec extends SpecificationWithJUnit {
       val orgSchedule = "R3/2012-01-01T00:00:00.000Z/P1D"
       val stream = new ScheduleStream(orgSchedule, null)
       stream.head must_==(orgSchedule, null, "")
-      stream.tail.get.head must_==("R2/2012-01-02T00:00:00.000Z/P1D", null, "")
-      stream.tail.get.tail.get.head must_==("R1/2012-01-03T00:00:00.000Z/P1D", null, "")
-      stream.tail.get.tail.get.tail.get.head must_==("R0/2012-01-04T00:00:00.000Z/P1D", null, "")
-      stream.tail.get.tail.get.tail.get.tail must_== None
+      stream.tail().get.head must_==("R2/2012-01-02T00:00:00.000Z/P1D", null, "")
```

Comment:
There's a note here on why we do this: http://docs.scala-lang.org/style/method-invocation.html